backend/apps/common/tests.py

```
@@ -575,14 +578,40 @@ def test_username_collision_resolution(self, mock_resolve):
         self.assertEqual(new_user.username, "juan.perez1")
 
     @patch("apps.common.views.resolve_user_from_request")
-    def test_existing_user_password_update(self, mock_resolve):
-        """If an existing user is provided with a password, it should be updated."""
+    @patch("apps.common.views.process_password_reset_request")
+    def test_existing_user_password_is_not_overwritten(self, mock_reset, mock_resolve):
+        self._mock_admin(mock_resolve)
+        old_password = "PASSWORD2"
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "old_password"

(S105)